### PR TITLE
Scoreboard 8

### PR DESCRIPTION
**Pull request by Marty99-dev**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindwar/game/solo/fragments/DemoFragment.kt [hunk 1]
```diff
@@ -83,22 +88,40 @@ class DemoFragment : Fragment() {
     ): View {
         val view = inflater.inflate(R.layout.activity_animated_demo, container, false)
 
+        // Get the scoreboard
+        scoreboard = view.findViewById(R.id.scoreboard)
+
+        // Create the adapter for the score board
+        scoreboardAdapter = ScoreboardAdapter(listOf("Marty", "Joris", "Nael", "Arthur", "Paul", "Henrique"))
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/game/solo/fragments/DemoFragment.kt:95]: It's probably temporary but there are hardcoded strings.

**Review by CHOCOLVT — APPROVED** [review]
great abstraction improvement

File: app/src/main/java/ch/epfl/sdp/blindwar/game/util/ScoreboardAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    // Return the size of your dataset (invoked by the layout manager)
+    override fun getItemCount() = dataSet.size
+
+    /**
+     * Increment the number of point of a player in the scoreboard
+     *
+     * @param playerName Name of the player
+     * @return
+     */
+    fun incrementPoint(playerName: String) {
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/game/util/ScoreboardAdapter.kt:50]: Maybe that's not really the play do that.

**Review by marluxiaboss — APPROVED** [review]
Good work! Some hardcoded string should probably be changed though.

**Pull request merged**